crates/agglayer-certificate-orchestrator/src/lib.rs

```diff
@@ -132,7 +133,7 @@ where
         clock: ClockRef,
         data_receiver: Receiver<(NetworkId, Height, CertificateId)>,
         cancellation_token: CancellationToken,
-        epoch_packing_task_builder: E,
+        epoch_packing_task_builder: Sc,
```

Comment:
Would this be a more appropriate name, given the generic bounds?
```suggestion
        settlement_client: Sc,
```